Sources/Testing/Running/Configuration.swift

```diff
@@ -20,6 +20,80 @@ public struct Configuration: Sendable {
   /// Whether or not to parallelize the execution of tests and test cases.
   public var isParallelizationEnabled = true
 
+  /// A type describing whether or not, and how, to iterate a test plan
+  /// repeatedly.
+  ///
+  /// When a ``Runner`` is run, it will run all tests in its corresponding
+  /// ``Runner/Plan`` according to the policy described by its
+  /// ``Configuration/iterationPolicy-swift.property`` property. For instance,
+  /// if that property is set to:
+  ///
+  /// ```swift
+  /// .repeating(.untilIssueRecorded, count: 10)
+  /// ```
+  ///
+  /// The entire test plan will be run repeatedly, up to 10 times. If an issue
+  /// is recorded, the current iteration will complete, but no further
+  /// iterations will be attempted.
+  ///
+  /// If the value of an instance's ``count`` property is `1`, the value of its
+  /// ``continuationCondition-swift.property`` property has no effect.
+  public struct IterationPolicy: Sendable {
```

Comment:
Bikeshedding a little, I'd like to propose we call this type "RepetitionPolicy", and refer to the general functionality with the base word "repetition", but still use the word "iteration" anywhere we need to refer to one individual unit of the repetition. So to me, that means these `Configuration` declarations would be phrased in terms of "repetition", while the additions to `Event` would continue using the word "iteration". Thoughts?

Reply:
I went back and forth on it but settled on "iteration" as a consistent term of art here. I'm not sure why we would want to say "repetition" here but "iteration" there—seems like it might be confusing and could imply two separate concepts. Do you feel strongly about changing it?

Reply:
It's been a few days since I looked at this code and ugh, I am not consistent at all between the two terms. I'll try to adopt what you suggest here.